### PR TITLE
Add a custom capability 'cloud:URL' and relevant documentation

### PR DESCRIPTION
**Pull request by mstrelex**
If cloud:URL is capability set, Agent (0.63.11+) will attempt to initialize a driver using provided URL of a cloud provider such as BrowserStack or SauceLabs.

### CODE REVIEW

**Comment by vitalybu** [comment]
How about adding SL to the unit tests we run as part of the CI?

File: README.md [hunk 1]
```diff
@@ -151,6 +151,23 @@ public ChromeDriver(final URL remoteAddress, final ChromeOptions options)
 
 It can also be set using the `TP_AGENT_URL` environment variable.
 
+## Remote (Cloud) Driver
+
+By default, TestProject Agent is using local Selenium and Appium servers. \
```
Comment (vitalybu) [inline README.md:156]:
```suggestion
By default, TestProject Agent communicates with the local Selenium or Appium server. \
```

**Comment by mstrelex** [comment]
> How about adding SL to the unit tests we run as part of the CI?

We can do it in a separate PR if we decide to do so and pass the URL+credentials via an environment variable.

File: README.md (hunk 1 above)
Comment (mstrelex) [reply README.md]: Fixed.

**Comment by vitalybu** [comment]
> We can do it in a separate PR if we decide to do so and pass the URL+credentials via an environment variable.

Why a separate PR?
IMO tests should be submitted with the functionality...

**Comment by mstrelex** [comment]
> > We can do it in a separate PR if we decide to do so and pass the URL+credentials via an environment variable.
> 
> Why a separate PR?
> IMO tests should be submitted with the functionality...

Added: faee77294247377935dfeb64c173c0370cc833ab

**Pull request merged**